apps/web/pages/api/auth/forgot-password.ts

```diff
@@ -0,0 +1,63 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { sendResetPassword } from "@documenso/lib/mail";
+import { defaultHandler, defaultResponder } from "@documenso/lib/server";
+import prisma from "@documenso/prisma";
+import crypto from "crypto";
+
+async function postHandler(req: NextApiRequest, res: NextApiResponse) {
+  const { email } = req.body;
+  const cleanEmail = email.toLowerCase();
+
+  if (!cleanEmail || !cleanEmail.includes("@")) {
```

Comment:
This would allow an email of `@` we can do a basic `/.+@.+/` regex instead which covers most usecases since the email rfc is cooked.